lib/realtime/rpc.ex

```diff
@@ -36,38 +36,29 @@ defmodule Realtime.Rpc do
   def enhanced_call(node, mod, func, args \\ [], opts \\ []) do
     timeout = Keyword.get(opts, :timeout, Application.get_env(:realtime, :rpc_timeout))
 
-    try do
-      :timer.tc(fn -> :erpc.call(node, mod, func, args, timeout) end)
-    catch
-      kind, reason ->
-        log_error("ErrorOnRpcCall", %{target: node, mod: mod, func: func, error: {kind, reason}})
-        {:error, "RPC call error"}
-    else
-      {_, {:EXIT, reason}} ->
-        log_error("ErrorOnRpcCall", %{target: node, mod: mod, func: func, error: {:EXIT, reason}},
-          mod: mod,
-          func: func,
-          target: node
-        )
+    with {latency, {status, _} = response} <-
+           :timer.tc(fn -> :erpc.call(node, mod, func, args, timeout) end) do
```

Comment:
is it safe that `erpc` can raise an error?

Reply:
I would expect the catch to handle those scenarios but still kind of leaning towards just using `:rpc`... wanted to avoid bigger code changes